src/System.Data.SqlClient/tests/FunctionalTests/CloneTests.cs

```diff
@@ -30,7 +30,7 @@ public void CloneSqlCommand()
 
             SqlCommand command = connection.CreateCommand();
             command.CommandText = "select 1";
-            command.CommandTimeout = 45;
+            command.ResetCommandTimeout();
```

Comment:
Can you put an assert here to verify that reset command timeout behaves as expected? Set the timeout value to something besides the default, then reset and check that the new value is the expected default.